code/modules/asset_cache/assets/plumbing.dm

```diff
@@ -17,7 +17,6 @@
 			"synthesizer",
 			"reaction_chamber",
 			"grinder_chemical",
-			"growing_vat",
```

Comment:
Why was this removed? The plumbing rcd UI needs this so does that still work?

Reply:
The science RPLD was removed. The growing vat can't be printed from the RPLD anymore. Also, that icon stage doesn't even exist in that file anymore; if it was still printable I'd a reference to the new icon, but it's not.